docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -164,9 +164,11 @@ Some possibly useful slots:
 
 - `appTitle`: allows you to customize the app title section in the layout header.
```

Comment:
Should we remove this since we are intending to deprecate `appTitle` on `DashboardLayout`?